pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -749,11 +749,6 @@ const (
 	// or the specified namespace was not present.
 	NamespaceCreatedByProjectController = "namespace.gardener.cloud/created-by-project-controller"
 
-	// DefaultVPNRange is the default IPv4 network range for the VPN between seed and shoot cluster.
-	DefaultVPNRange = "192.168.123.0/24"
-	// DefaultVPNRangeV6 is the default IPv6 network range for the VPN between seed and shoot cluster.
-	DefaultVPNRangeV6 = "fd8f:6d53:b97a:1::/120"
-
```

Comment:
I wanted to keep it similar to other default values, which are also constants in the versioned API package itself. In a new version of the API, we might apply different defaults, so it's probably sensible to put it into the versioned package.
I don't have a strong opinion on where to put them though, because both packages are used for constants in different places.

Reply:
Fine with me. I doubt that these ranges change a lot over time, though.